src/modules/auth/index.js

```diff
@@ -75,7 +75,7 @@ export default class AuthModule extends Module {
 				this.App.addAxiosInterceptor(this.authInterceptor());
 
 				// Authorization of the user based on rbac
-				if (this.Authorization?.Authorize) {
+				if (this.Authorization?.Authorize && this.App.Services.TenantService) {
```

Comment:
Nope, tenant service is irrelevant for user authorisation.

Reply:
Refactored in this commit: https://github.com/TeskaLabs/asab-webui/pull/105/commits/1732113a864b62f879e02d01aa5aed2eec6a7698